### ISSUE TITLE
Questiion about dataset and loss ?

### ISSUE THREAD

**Issue by brian3021** [noahcao/Pixel2Mesh#32]
Hi, thanks for sharing the repo.

I'm doing the research on 3D reconstruction from single image now, and I found your repo.

and I have seen that in `Shapenet.py`, you did `pts -= [0.0, 0.0, -0.8]`.
**why you give the offset to pts?**

and for loss, I saw you did this:  `1.0 * ( torch.mean(dist1) + 1.0 * torch.mean(dist2) )`
I have question because I saw the origin author did: `3000 * ( torch.mean(dist1) + 0.55 * torch.mean(dist2) )`
**So why you change these two factor, and how should I choose these two factor?**

Thx.

**Comment by brian3021** [comment]
> Hi, thanks for sharing the repo.
> 
> I'm doing the research on 3D reconstruction from single image now, and I found your repo.
> 
> and I have seen that in `Shapenet.py`, you did `pts -= [0.0, 0.0, -0.8]`.
> **why you give the offset to pts?**
> 
> and for loss, I saw you did this: `1.0 * ( torch.mean(dist1) + 1.0 * torch.mean(dist2) )`
> I have question because I saw the origin author did: `3000 * ( torch.mean(dist1) + 0.55 * torch.mean(dist2) )`
> **So why you change these two factor, and how should I choose these two factor?**
> 
> Thx.

I have seen the answer about mesh_pos in issue #16 . Thanks!!

And also seen the issue #15 for chamfer loss scale.
So just a simple follow-up, both the scale (3000 to all chamfer loss and 0.55 for dist2) from origin tensorflow implementation, 
you didn't use them, instead you change learning rate to 1e-4 and get performance on par.

So the conclusion is that, the scale is not necessary, right?
